src/insertion_sort.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""The following code implements an insertion_sort function designed
+to be used on a standart Python list."""
+from __future__ import unicode_literals
+import random
+import datetime
+
+
+def insertion_sort(sort_lst):
+    if not isinstance(sort_lst, list):
+        raise TypeError("Your input must be a Python list.")
+    sort_position = -1
+    while sort_position < len(sort_lst):
+        sort_position += 1
+        comp_indx = sort_position + 1
+        while comp_indx < len(sort_lst):
+            if sort_lst[comp_indx] < sort_lst[sort_position]:
+                sort_lst[comp_indx], sort_lst[sort_position] = sort_lst[sort_position], sort_lst[comp_indx]
```

Comment:
This isn't quite the correct implementation for insertion sort; it's a little slower as you are doing more value assignment statements than necessary.

What you want to do instead is assign the "value to be re-inserted" to a variable, holding it inside as you iterate down the list (in your while loop) until you find the right spot to re-insert the value. As you are iterating down the list in your while loop, just shift values up (one assignment statement) instead of swapping them (two assignment statements).